frontend/lib/src/components/shared/BaseButton/styled-components.ts

```diff
@@ -25,13 +25,15 @@ export enum BaseButtonKind {
   PRIMARY = "primary",
   SECONDARY = "secondary",
   TERTIARY = "tertiary",
+  GHOST = "ghost",
```

Comment:
👻 😄 
What's the visual difference between the "ghost" button and the tertiary button? It looks like the Ghost button is only used in the StreamlitDialog and I am wondering whether we could simply use the new tertiary button there? Not PR blocking but it might make sense to talk to @jrieke about streamlining this.

Reply:
Difference is pretty negligible after the last design pass - the ghost button (left) has padding that that tertiary (right) button does not. Like the idea of consolidating these and handling the spacing in dialogs separately if others aligned. 

<img width="200" alt="Screenshot 2024-12-03 at 7 06 46 PM" src="https://github.com/user-attachments/assets/672af587-9f8f-4216-bbf8-3bb194481f87"> <img width="125" alt="Screenshot 2024-12-03 at 7 07 04 PM" src="https://github.com/user-attachments/assets/2b8f113d-f62b-42a0-a86f-980491da272f">